AffiliateWP/affiliatewp-checkout-referrals
# issue/19 Fixes WooCommerce integration: when affiliate selection is required it no longer shows the word Optional

Issue #19


**Housekeeping**


**Proposed changes**


## Before 
<img width="662" alt="Screen Shot 2021-02-03 at 2 30 42 PM" src="https://user-images.githubusercontent.com/6202457/106803265-039dc000-6632-11eb-9bdd-e35555c213ca.png">

## After 
<img width="656" alt="Screen Shot 2021-02-03 at 3 11 25 PM" src="https://user-images.githubusercontent.com/6202457/106803342-1dd79e00-6632-11eb-8523-54917312fc79.png">

## Testing

AffiliateWP: 2.2.2
Checkout Referrals: 1.0.6
Integration: WooCommerce

1. Make sure the Required Affiliate Selection is checked under Affiliates -> Settings -> Checkout Referrals
2. Create a WooCommerce product if needed, add it to the cart, and go to the checkout page.
3. The Affiliate Selection label should show an asterisk and no longer show "(optional)" like the above before pic.
